site/src/modules/provisioners/ProvisionerGroup.tsx

```diff
@@ -343,6 +358,7 @@ const PskProvisionerTags: FC<PskProvisionerTagsProps> = ({ tags }) => {
 				css={{
 					"& .MuiPaper-root": {
 						padding: 20,
+						minWidth: "unset",
```

Comment:
Does this do anything? I thought `min-width` wasn't an inheritable property?

Reply:
it's not inherited, it's set by MUI